### PR TITLE
Winit version 0.30.0

### PR DESCRIPTION
**Pull request by kchibisov**
--

Should be fine after these:

- [x] https://github.com/rust-windowing/winit/pull/3653
- [x] https://github.com/rust-windowing/winit/pull/3654
- [x] https://github.com/rust-windowing/winit/pull/3597

### CODE REVIEW

File: src/changelog/unreleased.md
Comment (MarijnS95) [inline src/changelog/unreleased.md:1]: It's a tad unfortunate that we loose `git blame` info on this file, but it's probably not used very often anyway.

**Review by MarijnS95 — APPROVED** [review]

File: src/changelog/unreleased.md
Comment (kchibisov) [reply src/changelog/unreleased.md:1]: It was kind of broken even before when I was moving changelog entries around.

**Comment by linkmauve** [comment]
Reading the changelog I wrote #3656, so that the new gestures support more than just two platforms. I’ve found that the unit of the rotation gesture isn’t specified, the Wayland protocol exposes it in degrees but I don’t know what Apple does, it would be useful to specify that before the release.

**Comment by kchibisov** [comment]
The gestures are present for a very long time, like 2-3 years, it's just they were renamed a bit this release, so while specifying them is good, they were always meant for macOS only and the metrics are the same as in all other releases.

File: src/changelog/unreleased.md
Comment (madsmtm) [reply src/changelog/unreleased.md:1]: Would it help if you did it in two commits, and then rebase merge the PR?

**Comment by linkmauve** [comment]
Ah you’re right, they were already present in the previous release, but the unit is still unspecified [in the documentation](https://docs.rs/winit/latest/winit/event/enum.WindowEvent.html#variant.TouchpadRotate), so I still don’t know what it’s meant to be even on macOS. Or do you mean it’s a unit/event specific to macOS and shouldn’t be tried to be matched on other windowing systems?

File: src/changelog/unreleased.md
Comment (kchibisov) [reply src/changelog/unreleased.md:1]: It doesn't matter how it's done, because it's moved anyway.

**Review by daxpedda — APPROVED** [review]

**Comment by madsmtm** [comment]
The unit of rotation is documented in https://github.com/rust-windowing/winit/pull/3597 to be degrees.

**Review by madsmtm — APPROVED** [review]

**Pull request merged**